dojo/tools/bearer_cli/parser.py

```diff
@@ -46,6 +46,7 @@ def get_findings(self, file, test):
                     sast_source_line=bearerfinding["source"]["start"],
                     sast_source_file_path=bearerfinding["filename"],
                     vuln_id_from_tool=bearerfinding["id"],
+                    unique_id_from_tool=bearerfinding["fingerprint"],
```

Comment:
```suggestion
                    # the fingerprint is not constant over time, but because it's not used for dedupe it's safe and useful to set it
                    unique_id_from_tool=bearerfinding["fingerprint"],
```